package.json

```diff
@@ -246,6 +256,11 @@
                     "when": "view == aws.explorer",
                     "group": "z_externalLinks@3"
                 },
+                {
+                    "command": "aws.submitFeedback",
+                    "when": "view == aws.explorer",
+                    "group": "z_externalLinks@4"
```

Comment:
I'm not sure I like having this right next to the icon. I'd recommend choosing one.

Reply:
Can you expand on this? This is for adding the context menu item.

Reply:
Yeah, you have the icon next to the menu and the option in the menu. I'd pick one.

Reply:
Oh, this is the same as jb. I think the intent is that we can add an option to hide the icon in the future. This is the behavior of the built-in vs code feedback icon.

I'm not really concerned with the duplication, though. I do like that there is a text option because icons can be easily overlooked.